pyaedt/icepak.py

```diff
@@ -2719,6 +2719,92 @@ def assign_surface_material(self, obj, mat):
                 self.materials.surface_material_keys[mat.lower()] = sm
         return True
 
+    @pyaedt_function_handler()
+    def create_two_resistor_network_block_depr(self, object_name, power, rjb, rjc, placement):
+        """Function to create 2-Resistor network object.
+
+        .. deprecated:: 0.6.30
+            This method will be replaced by `create_two_resistor_network_block`.
+
+        Parameters
+        ----------
+        object_name : str
+            name of the object (3D block primitive) on which 2-R network is to be created
+        power : float
+            junction power in [W]
+        rjb : float
+            Junction to board thermal resistance in [K/W]
+        rjc : float
+            Junction to case thermal resistance in [K/W]
+        placement : str
+            Acceptable entries are "top" or "bottom"
+            "top" : network block is placed on top of board
+            "bottom" : network block is placed on bottom of board
```

Comment:
```suggestion
        power : float
            Junction power in [W].
        rjb : float
            Junction-to-board thermal resistance in [K/W].
        rjc : float
            Junction-to-case thermal resistance in [K/W].
        placement : str
            Placement of the network block. Options are:
            - ``top``: Network block is placed on top of the board.
            - "bottom" : Network block is placed on bottom of the board.
```
Can we just say Options are ``top`` and ``bottom``? The descriptions seem unnecessary.